python/cugraph/components/connectivity.py

```diff
@@ -48,3 +49,39 @@ def weakly_connected_components(G):
     df = connectivity_wrapper.weakly_connected_components(G.graph_ptr)
 
     return df
+
+
+def strongly_connected_components(G):
+    """
+    Generate the stronlgly connected components and attach a component label to
+    each vertex.
+
+    Parameters
+    ----------
+    G : cugraph.Graph
+      cuGraph graph descriptor, should contain the connectivity information as
+      an edge list (edge weights are not used for this algorithm). The graph
+      can be either directed or undirected where an undirected edge is
+      represented by a directed edge in both directions.
+      The adjacency list will be computed if not already present.
+      The number of vertices should fit into a 32b int.
+
+    Returns
+    -------
+    df : cudf.DataFrame
+      df['labels'][i] gives the label id of the i'th vertex
+      df['vertices'][i] gives the vertex id of the i'th vertex
+
+    Examples
+    --------
+    >>> M = read_mtx_file(graph_file)
+    >>> sources = cudf.Series(M.row)
+    >>> destinations = cudf.Series(M.col)
+    >>> G = cuGraph.Graph()
```

Comment:
cuGraph.Graph() should be cugraph.Graph()

This code implicitly assumes "import cugraph" (not "import cugraph as cuGraph") is executed before this example.

Reply:
Fixed.